cmake/DD4hepBuild.cmake

```
@@ -649,16 +649,16 @@ endfunction(dd4hep_add_plugin)
 macro(DD4HEP_SETUP_ROOT_TARGETS)
 
   #Check if Python version detected matches the version used to build ROOT
+  SET(Python_FIND_FRAMEWORK LAST)
   IF(${ROOT_VERSION} VERSION_GREATER_EQUAL 6.19)
```

We need to check if root was built with python, right?
```suggestion
  IF((TARGET ROOT::PyROOT or TARGET ROOT::ROOTTPython) AND ${ROOT_VERSION} VERSION_GREATER_EQUAL 6.19)
```

---

This additional check is a nice idea, but need to be separate `IF`, because the `${ROOT_VERSION}` determines how to call `find_package` for old versions of ROOT.

---

I don't think it needs to be separate, because for older versions of root, we just look for any python in any case, right?
We don't need to fail because there isn't any pyroot